frontend/src/components/profile/components/edite-document-popup/edite-document-popup.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { IDocument } from 'common/interfaces';
+import { Modal, Button, Radio } from 'components/common';
+import { ButtonColor, ButtonStyleType, DocumentStatus, ButtonType } from 'common/enums';
+import { BindingCb } from 'common/types';
+import { useForm } from 'react-hook-form';
+import { documentStatusToReadable, EditeDocumentCb, DocumentFormData } from '../../common';
+import styles from './styles.module.scss';
+
+type Props = {
+  document: IDocument;
+  isShow:boolean;
+  onToggleModal:BindingCb;
+  onEditeDocument:EditeDocumentCb;
+};
+
+const verifyDocumentOptions = [
+  {
+    label: documentStatusToReadable[DocumentStatus.VERIFIED],
+    value: DocumentStatus.VERIFIED,
+  },
+  {
+    label: documentStatusToReadable[DocumentStatus.IN_REVIEW],
+    value: DocumentStatus.IN_REVIEW,
+  },
+];
+
+const EditeDocumentPopup: React.FC<Props> = ({ document, isShow, onToggleModal, onEditeDocument }) => {
+  const [radioValue, setRadioValue] = React.useState<DocumentStatus>(document.status);
+  const { register, handleSubmit, watch } = useForm<DocumentFormData>();
+
+  const handleChangeRadio = () => {
+    const { status } = watch();
+    setRadioValue(status);
+  };
+  const handleSubmitForm = (formData:DocumentFormData) => {
+    const update:IDocument = { ...document, status: formData.status };
+    onEditeDocument(update);
+  };
+  return (
+    <Modal isShow={isShow}>
+      <div className={styles.formContainer}>
+        <button className={styles.closeButton} onClick={onToggleModal} type="button">
+          &#10060;
```

Comment:
Add correct label
use `visually-hidden` class, if we don't have a label in the markup. See **A12**

Reply:
Missed